### PR TITLE
Inclusion of a configuration file in JSON format

### PR DESCRIPTION
**Pull request by gtorodelvalle**
- 100% tests passed
- Implements https://github.com/telefonicaid/IoT-STH/issues/78
- Assigned to @frbattid

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -274,15 +274,23 @@ The STH component server is ready to be started.
...
+version of the STH component is used.
+* The [`config.js`](https://github.com/telefonicaid/IoT-STH/blob/develop/config.js) located at the root of the STH component code, a JSON formatted file including the configuration properties.
+
+It is important to note that environment variables, if set, take precedence over the properties defined in the `config.js` file.
+
 The script accepts the following parameters as environment variables:
 
 - STH_HOST: The host where the STH server will be started. Optional. Default value: "localhost".
-- STH_PORT: The port where the STH server will be listening. Optional. Default value: 8666.
+- STH_PORT: The port where the STH server will be listening. Optional. Default value: "8666".
```
Comment (frbattid) [inline README.md]: Just a suggestion for the future, not this PR: why not creating a table having "parameter", "description", "mandatory", "default value" columns.

**Comment by frbattid** [comment]
LGTM with a minor comment not to be taken into account in this PR :)

I think a rebase is needed!

File: config.js [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+  //  (which creates a MongoDB collection per service path and entity to store the data) and
+  //  collection-per-attribute (which creates a collection per service path, entity and attribute
+  //  to store the data). Default value: "collection-per-entity".
+  dataModel: 'collection-per-entity'
+};
+
+// Logging configuration
+//------------------------
+config.logging = {
+  // The logging level of the messages. Messages with a level equal or superior to this will be logged.
```
Comment (iariasleon) [inline config.js]: append allowed values, ex: trace, debug, etc

File: README.md (hunk 1 above)
Comment (gtorodelvalle) [inline README.md]: Issue https://github.com/telefonicaid/IoT-STH/issues/82 created to track this ;) Thanks @frbattid !

File: config.js (hunk 2 above)
Comment (gtorodelvalle) [inline config.js]: Included in https://github.com/telefonicaid/IoT-STH/commit/ad710c652c875956adb26f1b4e1c412ab974c2fa Thanks @iariasleon ! ;)

**Pull request merged**